project/Dependencies.scala

```diff
@@ -132,6 +132,9 @@ object Dependencies {
 
       val junit = Compile.junit % "optional;provided;test"
 
+
+      val scalatest = Def.setting { "org.scalatest" %% "scalatest" % scalaTestVersion.value % "optional;provided;test" } // ApacheV2
```

Comment:
dep / provided looks good... 👍 Did we sanity check what ends up in POM so we don't accidentally do it wrong?
AFAIR some of the scopes sometimes work weird...

Reply:
it becomes:
```
        <dependency>
            <groupId>junit</groupId>
            <artifactId>junit</artifactId>
            <version>4.12</version>
            <scope>test</scope>
            <optional>true</optional>
        </dependency>
        <dependency>
            <groupId>org.scalatest</groupId>
            <artifactId>scalatest_2.12</artifactId>
            <version>3.0.5</version>
            <scope>test</scope>
            <optional>true</optional>
        </dependency>
```

which I guess is fine